README.md

```diff
@@ -49,6 +48,56 @@ The tool has built in help that can be accessed by hitting the `?` key.
 |  `f`         | Set selected commit to be fixed-up |
 |  `d`         | Set selected commit to be dropped |
 
+
+## Development
+
+### Install Rust
+
+To hack at the project you will need to [install Rust](https://doc.rust-lang.org/book/getting-started.html). This
+can generally be done using the rustup script:
+
+***Note:** Debian and similar does not install curl by default: install with `sudo apt-get install curl`*
+
+     curl https://sh.rustup.rs -sSf | sh
```

Comment:
The rustup recommended install procedure is likely to change in the future, so it might be better to omit this, and just link the `rustup` project or website or something.